tests/dummy/app/styles/app.scss

```diff
@@ -1,6 +1,5 @@
-@import 'compass-boilerplate-vendor/normalize';
-@import 'compass-boilerplate-vendor/html5bp';
-@import 'compass-boilerplate';
+@import 'vendor/normalize';
+@import 'vendor/html5bp';
```

Comment:
As a quick fix, I have moved the `normalize` and `html5bp` files into this project. These files are added in this PR below.